.github/workflows/make-installer-linux.yml

```diff
@@ -75,6 +78,15 @@ jobs:
             -DCMAKE_INSTALL_PREFIX=/usr
             -DCPACK_GENERATOR="TGZ;DEB"
           cmake_commands: '--target package'
+      - name: 'Rename assets'
+        if: ${{ inputs.artifact_version_suffix != '' }}
+        shell: bash
+        run: |
+          for f in build/packages/xournalpp-*; do
+            correct_name="$(echo "$f" | sed "s/xournalpp-\(\(\:\?\(\:\?[0-9]\+\.\)\+[0-9]\+\)\(\:\?[+~][^-]\+\)\?\)-/xournalpp-\1${{ inputs.artifact_version_suffix }}-/")"
```

Comment:
Please add an example (about what $correct_name results from which $f and $inputs.artifact_version_suffix) as a comment for this complicated expression.

Reply:
Yes, like that.